README.md

```diff
@@ -49,6 +49,11 @@ Run all unittests and collect the code coverage:
 (env) $ coverage run --source="./src/qilib" -m unittest discover -s src/tests -t src
 (env) $ coverage report -m
 ```
+###Type Checking
+For static type checking, mypy can be used
+```
+(env) mypy --strict --ignore-missing-imports --allow-untyped-calls -p qilib
```

Comment:
in setup.py, add mypy to extras_require for dev-install so:

      extras_require={
          'dev': ['pytest>=3.3.1', 'coverage>=4.5.1', 'mongomock', 'mypy'],

Reply:
in memory.py unused 'List' type